contracts/UmaConditionalTokensBinaryAdapter.sol

```diff
@@ -290,90 +159,73 @@ contract UmaConditionalTokensBinaryAdapter is Auth, ReentrancyGuard {
         if (!isQuestionInitialized(questionID)) {
             return false;
         }
+
         QuestionData storage questionData = questions[questionID];
-        // Ensure resolution data has been requested for question
-        if (resolutionDataRequested(questionData) == false) {
-            return false;
-        }
+
         // Ensure question has not been resolved
         if (questionData.resolved == true) {
             return false;
         }
+
         // Ensure question has not been settled
         if (questionData.settled != 0) {
             return false;
         }
 
-        OptimisticOracleInterface optimisticOracle = getOptimisticOracle();
+        // If the question is disputed by the DVM, do not wait for DVM resolution
+        // instead, immediately flag the question as ready to be settled
+        if (_isDisputed(questionData)) {
+            return true;
+        }
 
+        // Check that the OO has an available price
         return
             optimisticOracle.hasPrice(
```

Comment:
the OO will only have a price after the liveness/challenge period has passed correct?

Reply:
Yeah, OO only has a price after liveness